contracts/token/WithdrawTokens.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/IERC721Upgradeable.sol";
+
+/**
+ * @title Making a token withdrawable by the owner to a specified onlyOwner setable address
```

Comment:
suggestion: split into seperate contracts.

By keeping the ERC20 and ERC721 as seperate sweeping contracts, it'll make testing easier.
In the future there'll likely be additional standards to support, to keep it convenient for the children, all of the sweeping contracts can be brought together allowing the children to import just one (but keeping discrete contracts for easier testing and encapsulation of behaviour).